### ISSUE TITLE
[hebr/he] There are 13 not 11 code points for vowel diacritics

### ISSUE THREAD

**Issue by bdenckla** [r12a/scripts#134]
[source] https://r12a.github.io/scripts/hebr/he

The page currently says (in two places):

> There are 11 vowel diacritics.

It also says:

> These are the niqqud used for modern Hebrew.

Followed by a list of 11 code points.

That list of 11 code points is missing the following 2 code points:

* U+05BA: HEBREW POINT HOLAM HASER FOR VAV
* U+05C7: HEBREW POINT QAMATS QATAN

A minor point is that the "modern" qualification in "These are the niqqud used for modern Hebrew" could be misleading. These are the _niqqud_ used for Biblical Hebrew as well, and indeed are the _niqqud_ used for all types of Hebrew of which I am aware. (Modern and Biblical Hebrew are the two main types of Hebrew but in some contexts, other "Hebrews" are distinguished such as Mishnaic Hebrew and liturgical Hebrew.)

This unnecessary (and therefore possibly misleading) qualification is also present in the statement:

> The Modern Israeli Hebrew alphabet has 22 letters, plus 5 word-final letters that have their own code points.

This 22+5 alphabet is used for Biblical Hebrew as well, and indeed is used for all types of Hebrew of which I am aware.

**Comment by r12a** [comment]
@bdenckla many thanks for raising these issues.  I have begun addressing them, but it may take a few days, since i fit this stuff in outside work hours.  I'll reply to your points as i make changes.

**Comment by r12a** [comment]
See https://github.com/r12a/scripts/issues/137 for the 2 code points mentioned.

I made some slight adjustments to the wording wrt the 'modern qualification', but essentially the page is describing what the modern israeli hebrew orthography uses, and is not concerned much with what other styles of Hebrew use.

**Comment by bdenckla** [comment]
As I mention in #137, I'm not sure why you are confident that HHFV & QQ are not used to point modern Hebrew. I have not had time to research this question one way or another. But it would make perfect sense to use them to point modern Hebrew. They reflect real sound differences that would otherwise have to be inferred, which is kind of the whole point of pointing modern Hebrew. In the case of HHFV the sound difference is not only real but dramatic: vo versus o!

So it would be make perfect sense to use HHFV & QQ to point modern Hebrew. Note that these cases are unlike the 3 _ḥataf_ vowel cases, which are often used in pointing modern Hebrew, but make no difference in pronunciation in any dialect of modern Hebrew of which I am aware!

Bottom line, without evidence one way or another, the safest thing to do, IMO, is to say that there are 13 points that COULD be used to point modern Hebrew, without passing judgment on how often particular ones of them are used. To some extent the goal of this page is to describe what facilities Unicode has that can be used to notate modern Hebrew. HHFV & QQ are definitely facilities Unicode has that can be used to notate modern Hebrew. How often people currently take advantage of those facilities is a separate question perhaps best left unjudged by this page.
